doc/source/changelog.rst

```diff
@@ -1,7 +1,26 @@
 Changelog
 ******************************
+
+- **v0.8.0** (*Unreleased*):
```

Comment:
Are the issues for the fixed bugs ever listed here in the `CHANGELOG`? Seems like I've seen that before, but maybe only for projects with a dedicated external bug tracker? Not a big deal, was just thinking it'd (perhaps) be nice to directly reference here to facilitate looking back, in case something comes up again and we intuit that it's happened before. Anyway, just a thought--not sure how beneficial and/or standard that is.

Reply:
Sorry, by issues I mean links to the issue page here, if applicable.

Reply:
I think yes, if it's something that is user-facing. that's been my rule-of-thumb... 

But what exactly do you mean? The bugs fixed *are* listed in the changelog... are you suggesting they be removed? I can't quite understand your suggestion.

Reply:
I meant a link to the GitHub issue

Reply:
I see. I guess I don't know that it's worth the work since it's more developer oriented, like you say...